app/actions/websocket/users.ts

```diff
@@ -22,49 +22,60 @@ import type UserModel from '@typings/database/models/servers/user';
 const {SERVER: {CHANNEL, CHANNEL_MEMBERSHIP}} = MM_TABLES;
 
 export async function handleUserUpdatedEvent(serverUrl: string, msg: any) {
-    const database = DatabaseManager.serverDatabases[serverUrl];
-    if (!database) {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
```

Comment:
Why don't we just do this ? 
```
    const database = DatabaseManager.serverDatabases[serverUrl]?.database;
```

Reply:
Operator is used, we could deconstruct it and get the database from the operator object to simplify